### DISCUSSION TITLE
dashboard not working

### DISCUSSION THREAD

**Discussion by Giuseppe-P** [dwainscheeren/dwains-lovelace-dashboard#281] [General]
after the update to home assistant 2021.4.4 the dashboard no longer works. How can I solve?

**Comment by AalianKhan** [comment]
Have you updated the dashboard to version 2.0.2.
Version 2021.4 introduced since new requirements for custom components so you must update to the latest version of Dwain dashboard

**Reply by Giuseppe-P** [reply]
Thanks it worked. I believed that updates were notified in home assistant.

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
